connector/spanmetricsconnector/connector.go

```diff
@@ -361,6 +361,9 @@ func (p *connectorImp) aggregateMetrics(traces ptrace.Traces) {
 							p.metricKeyToDimensions.Add(eKey, eAttributes)
 						}
 						e := events.GetOrCreate(eKey, eAttributes)
+						if p.config.Exemplars.Enabled && !span.TraceID().IsEmpty() {
```

Comment:
is there a test for this?

Reply:
@codeboten Yes, the function in `connector_test.go` is **TestExemplarsForSumMetrics**